src/ONE.Models.CSharp/Enums/EnumLogLevel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ONE.Models.CSharp.Enums
+{
+    public enum EnumLogLevel
+    {
+        Unknown = 0,
+        Fatal = 1,
+        Error = 2,
+        Warn = 3,
+        Info = 4,
+        Debug = 5,
+        Trace = 6
+    }
```

Comment:
Doesn't really matter, but you can omit the numbers after the 1st one and it will do the counting for you:
```suggestion
    public enum EnumLogLevel
    {
        Unknown = 0,
        Fatal,
        Error,
        Warn,
        Info,
        Debug,
        Trace
    }
```

Reply:
Noted